www/js/client.js

```diff
@@ -374,7 +374,8 @@ window.filesender.client = {
                 return uxhr;
             }
         };
-        
+
+        var chunkid = Math.floor(offset / window.filesender.config.upload_chunk_size);
```

Comment:
Please cap it off to ensure no more than 2^32-1 chunks are sent, or if 2^32 are sent that at least the last one is not a full chunk.  This is vital for the security of the encryption, as stated on Page 19:

  * It is vital for counter mode encryption that the 32-bit counter will never ever wrap around.

Reply:
Take note that this means a maximum file size of 16 * 2^32 - 1 -- in light of chunks being the 16-byte strings that pass through AES.